client/src/components/DataFiles/DataFiles.jsx

```diff
@@ -143,10 +145,14 @@ const DataFiles = () => {
     );
   }
 
+  if (listingParams.system === noPHI) {
+    label = 'UNPROTECTED';
+  }
+  
   return (
     <Section
       bodyClassName="has-loaded-datafiles"
-      introMessageName="DATA"
+      introMessageName={label}
```

Comment:
No change needed, but FYI this could be a one-liner:
```suggestion
      introMessageName={listingParams.system === noPHI ? "UNPROTECTED" : "DATA"}
```

Reply:
I went ahead and added it anyway with the other changes.